packages/custom-calendar-builder/src/module.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Calendar Builder Module for Seasons & Stars
+ * Provides a user interface for creating and editing custom calendar JSON files
+ */
+
+import { CalendarBuilderApp } from './calendar-builder-app';
+
+let calendarBuilderApp: CalendarBuilderApp | null = null;
+
+/**
+ * Initialize the Calendar Builder module
+ */
+Hooks.once('init', () => {
+  console.log('Calendar Builder | Initializing');
+
+  // Register the calendar builder application
+  calendarBuilderApp = new CalendarBuilderApp();
+
+  // Add hook for core module to provide integration points
+  Hooks.on('seasons-and-stars.registerCalendarBuilderIntegration', (integration: any) => {
+    console.log('Calendar Builder | Registering integration from core module', integration);
+    if (calendarBuilderApp) {
+      calendarBuilderApp.registerIntegration(integration);
+    }
+  });
```

Comment:
This feels like something that could just be done via the public API of the core module (for example, `game.seasonsAndStars.validateCalendar(...)` or similar).